packages/cypher-builder/src/expressions/procedures/db.ts

```diff
@@ -58,14 +59,20 @@ export class FullTextQueryNodes extends Clause {
 
     public getCypher(env: CypherEnvironment): string {
         const targetId = env.getReferenceId(this.targetNode);
+        let scoreYield = "";
+        if (this.scoreVar) {
+            const scoreId = env.getReferenceId(this.scoreVar);
+            scoreYield = `, score AS ${scoreId}`;
+        }
+
+        const textSearchStr = `CALL db.index.fulltext.queryNodes(
+            "${this.indexName}",
+            ${this.phrase.getCypher(env)}
+        ) YIELD node AS ${targetId}${scoreYield}`;
```

Comment:
After some discussion, Darrell and I decided to go with a one-liner for this, so:

```suggestion
        const textSearchStr = `CALL db.index.fulltext.queryNodes("${this.indexName}", ${this.phrase.getCypher(env)}) YIELD node AS ${targetId}${scoreYield}`;
```